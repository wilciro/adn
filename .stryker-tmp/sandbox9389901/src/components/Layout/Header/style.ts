// @ts-nocheck
function stryNS_9fa48() {
  var g = new Function("return this")();
  var ns = g.__stryker__ || (g.__stryker__ = {});

  if (ns.activeMutant === undefined && g.process && g.process.env && g.process.env.__STRYKER_ACTIVE_MUTANT__) {
    ns.activeMutant = g.process.env.__STRYKER_ACTIVE_MUTANT__;
  }

  function retrieveNS() {
    return ns;
  }

  stryNS_9fa48 = retrieveNS;
  return retrieveNS();
}

stryNS_9fa48();

function stryCov_9fa48() {
  var ns = stryNS_9fa48();
  var cov = ns.mutantCoverage || (ns.mutantCoverage = {
    static: {},
    perTest: {}
  });

  function cover() {
    var c = cov.static;

    if (ns.currentTestId) {
      c = cov.perTest[ns.currentTestId] = cov.perTest[ns.currentTestId] || {};
    }

    var a = arguments;

    for (var i = 0; i < a.length; i++) {
      c[a[i]] = (c[a[i]] || 0) + 1;
    }
  }

  stryCov_9fa48 = cover;
  cover.apply(null, arguments);
}

function stryMutAct_9fa48(id) {
  var ns = stryNS_9fa48();

  function isActive(id) {
    if (ns.activeMutant === id) {
      if (ns.hitCount !== void 0 && ++ns.hitCount > ns.hitLimit) {
        throw new Error('Stryker: Hit count limit reached (' + ns.hitCount + ')');
      }

      return true;
    }

    return false;
  }

  stryMutAct_9fa48 = isActive;
  return isActive(id);
}

import styled from 'styled-components';
export const Header = stryMutAct_9fa48("99") ? styled.header`` : (stryCov_9fa48("99"), styled.header`
  width: 90%;
  padding: 10px 15px;
  border-radius: 10px;
  box-shadow: 0 0 2px rgba(10, 10, 10, 0.5);
  background: rgba(240, 240, 240, 0.7);
  backdrop-filter: blur(5px);
  margin: auto;
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: fixed;
  top: 5px;
  left: 5%;
  z-index: 5;

  @media (max-width: 768px) {
    width: 100%;
    left: 0px;
    top: 0px;
  }
`);
export const ButtonsSeparator = stryMutAct_9fa48("100") ? styled.div`` : (stryCov_9fa48("100"), styled.div`
  display: flex;
  align-items: center;
  gap: 10px;
`);
interface NavProps {
  open: boolean;
}
export const Nav = stryMutAct_9fa48("101") ? styled.nav<NavProps>`` : (stryCov_9fa48("101"), styled.nav<NavProps>`
  width: 100%;
  height: 100vh;
  position: fixed;
  top: 0px;
  background: rgba(10, 10, 10, 0.8);
  backdrop-filter: blur(10px);
  z-index: 10;
  display: ${stryMutAct_9fa48("102") ? () => undefined : (stryCov_9fa48("102"), p => p.open ? stryMutAct_9fa48("103") ? "" : (stryCov_9fa48("103"), 'flex') : stryMutAct_9fa48("104") ? "" : (stryCov_9fa48("104"), 'none'))};
  align-items: center;
`);
export const NavMenu = stryMutAct_9fa48("105") ? styled.ul`` : (stryCov_9fa48("105"), styled.ul`
  list-style: none;
  font-size: 8vw;
  margin-left: 5%;

  a {
    color: white;
    text-decoration: none;
    transition: all 0.3s;
    &:hover {
      opacity: 0.9;
      color: var(--primary-color);
    }
  }
`);
export const NavClose = stryMutAct_9fa48("106") ? styled.button`` : (stryCov_9fa48("106"), styled.button`
  position: fixed;
  top: 10px;
  right: 10px;
  font-size: 4em;
  width: 100px;
  height: 100px;
  border-radius: 100%;
  cursor: pointer;
  background: none;
  color: white;
  border: none;
  &:hover {
    opacity: 0.8;
    color: var(--primary-color);
  }
`);