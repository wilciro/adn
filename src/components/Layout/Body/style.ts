import styled from 'styled-components';

interface BodyProps {
  dark: boolean;
}

export const Body = styled.div<BodyProps>`
  width: 80%;
  padding: 80px 15px;
  border-radius: 10px;
  margin: 20px auto;
  min-height: 100vh;

  @media (max-width: 768px) {
    width: 100%;
  }
`;
