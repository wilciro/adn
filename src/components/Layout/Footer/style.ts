import styled from "styled-components";

export const Footer = styled.header`
    width: 90%;
    padding: 10px 15px;
    margin: 20px auto;
    text-align: center;
`;
