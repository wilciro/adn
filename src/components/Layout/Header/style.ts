import styled from 'styled-components';

export const Header = styled.header`
  width: 90%;
  padding: 10px 15px;
  border-radius: 10px;
  box-shadow: 0 0 2px rgba(10, 10, 10, 0.5);
  background: rgba(240, 240, 240, 0.7);
  backdrop-filter: blur(5px);
  margin: auto;
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: fixed;
  top: 5px;
  left: 5%;
  z-index: 5;

  @media (max-width: 768px) {
    width: 100%;
    left: 0px;
    top: 0px;
  }
`;

export const ButtonsSeparator = styled.div`
  display: flex;
  align-items: center;
  gap: 10px;
`;

interface NavProps {
  open: boolean;
}

export const Nav = styled.nav<NavProps>`
  width: 100%;
  height: 100vh;
  position: fixed;
  top: 0px;
  background: rgba(10, 10, 10, 0.8);
  backdrop-filter: blur(10px);
  z-index: 10;
  display: ${(p) => (p.open ? 'flex' : 'none')};
  align-items: center;
`;

export const NavMenu = styled.ul`
  list-style: none;
  font-size: 8vw;
  margin-left: 5%;

  a {
    color: white;
    text-decoration: none;
    transition: all 0.3s;
    &:hover {
      opacity: 0.9;
      color: var(--primary-color);
    }
  }
`;

export const NavClose = styled.button`
  position: fixed;
  top: 10px;
  right: 10px;
  font-size: 4em;
  width: 100px;
  height: 100px;
  border-radius: 100%;
  cursor: pointer;
  background: none;
  color: white;
  border: none;
  &:hover {
    opacity: 0.8;
    color: var(--primary-color);
  }
`;
