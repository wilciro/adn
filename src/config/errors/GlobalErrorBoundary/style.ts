import styled from "styled-components";

export const Container = styled.div`
    width: 60%;
    padding: 10px 15px;
    margin: 20px auto;
    text-align: center;
    height: 100vh;
    display: flex;
    align-items: center;
`;