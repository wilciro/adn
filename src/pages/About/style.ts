import styled from 'styled-components';

export const AboutDiv = styled.section`
  width: 96%;
  margin: auto;
  display: flex;
  justify-content: space-between;
  align-items: center;

  img {
    width: 40%;
  }

  @media (max-width: 768px) {
    width: 100%;
    display: block;
    img {
      width: 70%;
      margin: auto;
      display: block;
    }
  }
`;
