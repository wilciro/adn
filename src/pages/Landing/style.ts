import styled from 'styled-components';

export const CheckoutDiv = styled.section`
  width: 300px;
  margin: auto;
  border: solid 1px rgb(230, 230, 230);
  border-radius: 10px;
  padding: 5px 25px;
  position: fixed;
  right: 10px;
  top: 100px;
  z-index: 10;
  background: white;

  @media (max-width: 1000px) {
    width: 100%;
    display: block;
    bottom: 0px;
    top: auto;
    right: auto;
    left: 0px;
    z-index: 3;
    background: rgba(255, 255, 255, 0.6);
    backdrop-filter: blur(5px);
  }
`;
