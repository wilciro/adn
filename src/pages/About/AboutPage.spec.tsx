import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import AboutPage from '.';
import { AboutDiv } from './style';

describe('AboutPage tests', () => {
  it('should match style', () => {
    const { container } = render(<AboutDiv>Ceiba</AboutDiv>);
    expect(container).toMatchSnapshot();
  });
  it('should match snapshot AboutPage', () => {
    const { container } = render(
      <BrowserRouter>
        <AboutPage />
      </BrowserRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
