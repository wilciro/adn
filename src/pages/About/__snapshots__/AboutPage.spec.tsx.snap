// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage tests should match snapshot AboutPage 1`] = `
<div>
  <section
    class="sc-bczRLJ dYjPoo"
  >
    <img
      alt="logo of vetapp"
      src="/assets/image.png"
    />
    <div>
      <h2>
        Sobre Vet App
      </h2>
      <p>
        VetApp es una aplicación desarrollada como proceso de inducción para la empresa Ceiba Software.
      </p>
      <p>
        VetApp está diseñado para permitir el agendamiento de citas médicas veterinarias mostrando su precio en tiempo real dependiendo del tipo de mascota a valorar.
      </p>
      <a
        href="/"
      >
        <button
          class="mantine-Button-filled mantine-Button-root mantine-1grzg0q"
          type="button"
        >
          <div
            class="mantine-3xbgk5 mantine-Button-inner"
          >
            <span
              class="mantine-qo1k2 mantine-Button-label"
            >
              Agendar cita
            </span>
          </div>
        </button>
      </a>
    </div>
  </section>
</div>
`;

exports[`AboutPage tests should match style 1`] = `
<div>
  <section
    class="sc-bczRLJ dYjPoo"
  >
    Ceiba
  </section>
</div>
`;
